test/unit/presenters/local_transaction_presenter_test.rb

```diff
@@ -16,4 +24,37 @@ def subject(content_item)
   test "#need_to_know" do
     assert_equal "foo", subject(details: { need_to_know: "foo" }).need_to_know
   end
+
+  test "#scotland_availability" do
+    assert_equal "devolved_administration_service", subject(@content_item_with_devolved_administration_availability, "Scotland").scotland_availability["type"]
+    assert_equal "https://www.gov.scot/stray-dog", subject(@content_item_with_devolved_administration_availability, "Scotland").scotland_availability["alternative_url"]
```

Comment:
You can probably make this test self-contained like the ones that precede it - which makes it easier to understand.

```suggestion
    devolved_administration = { "type" => "devolved_administration_service", "alternative_url" => "https://gov.scot" }
    assert_equal devolved_administration, subject(details: { scotland_availability: devolved_administration })
```

Reply:
Ooh I did that as I wanted to make use of the content-schema example... but I suppose defining the structure in these tests saves someone having to go and find the schema. Have updated!